src/models/leaderboard.rs

```diff
@@ -0,0 +1,45 @@
+use actix_web::web;
+use charybdis::callbacks::Callbacks;
+use charybdis::types::{Frozen, Int, Set, Text, Timestamp, Uuid};
+use charybdis_macros::charybdis_model;

```

Comment:
Sorry for this as docs were wrong. We just need `use charybdis::macros::charybdis_model;`  so we don't have to install `charybdis_macros` separately.